admin/category_product_listing.php

```diff
@@ -816,17 +820,17 @@
 // Split Page
 
 // reset page when page is unknown
-            if ((empty($_GET['page']) || $_GET['page'] == '1') && !empty($_GET['pID'])) {
+            if ((empty($_GET['page']) || $_GET['page'] === 1) && !empty($_GET['pID'])) {
               $check_page = $db->Execute($products_query_raw);
               if ($check_page->RecordCount() > $max_results) {
                 $check_count = 0;
                 foreach ($check_page as $item) {
-                  if ($item['products_id'] == $_GET['pID']) {
+                  if ($item['products_id'] === $_GET['pID']) {
```

Comment:
`$_GET['pID']` gets sanitized to an integer; the exactly-equal-to compare will never 'hit'.